documentation/modules/proc-enabling-tls.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-enabling-tls-{context}']
+= Accessing the cluster
+
+As `loadbalancer` is used for external access to the cluster, a cluster CA certificate is required to enable TLS (Transport Layer Security) encryption between the broker and the client.
+
+.Prerequisites
+
+* A Kafka cluster running within the Kubernetes cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Find the IP address of the bootstrap `loadbalancer`:
++
+[source, shell, subs=+quotes]
+----
+kubectl get service my-cluster-kafka-external-bootstrap -o=jsonpath='{.status.loadBalancer.ingress[0].ip}{"\n"}'
+----
++
+Use the IP address together with port 9094 in your Kafka client as the bootstrap address.
+
+. Extract the public certificate of the broker certification authority:
++
+[source, shell, subs=+quotes]
+----
+kubectl get secret my-cluster-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
+----
+
+. Import the trusted certificate to a truststore:
++
+[source, shell, subs=+quotes]
+----
+keytool -keystore client.truststore.jks -alias CARoot -import -file ca.crt
+----
++
+You are now ready to start sending and receiving messages.
```

Comment:
You need to configure the client to use that truststore.

Reply:
@tombentley Are you suggesting we just mention that here or do we need to change what we describe in the proc-sending-messages.adoc, where we have the truststore parameters?

Reply:
The problem here is you have two separate procedures ("Accessing the cluster" and "Sending and receiving messages from a topic") which, from the PoV of someone trying Strimzi for the first time, should really be a single procedure. By the end of "Accessing the cluster" the user is not actually able to _access the cluster_ they merely have the prerequisite information necessary for that.

I can see the value of having separate procedures, since only need to get those prereqs once. But I think you need to make clear in `Sending and receiving messages from a topic` that the load-balancer address and truststore came from the previous proc. I know you've said "Ensure you have the prerequisites and then follow the tasks in the order provided in this chapter." as the start of the section, but that's really a cheat -- if modules are supposed to be reusable (i.e. standalone) then you can't rely on that and you should have something in the Prerequisites of each procedure as well/instead.

Reply:
Thanks for clarifying.

So in **_Accessing the cluster_**, we now bookend the steps with:

START: In this procedure you obtain a loadbalancer address and truststore, which are used to access your cluster.
END: You now have the information needed to start sending and receiving messages.

In **_Sending and Receiving_** we have:

.Prerequisites
* xref:proc-enabling-tls-{context}[Loadbalancer address and truststore required for access]

Reply:
:+1: